### ISSUE TITLE
Repeated keyword arguments are not detected

### ISSUE THREAD

**Issue by dandv** [microsoft/pyright#8260]
## Environment data

- Pylance version: v2023.10.40
- OS and version: Fedora Linux 38
- Python version 3.12.3
- Cursor v0.35.1

## Code Snippet

```python
print(
    'Hello',
    end = ' ',
    end = 'foo'
)

print('Pylance does work'
```

## Expected behavior

Screenshot of the same file in PyCharm:

![image](https://github.com/microsoft/pylance-release/assets/33569/f3f3f44b-1654-428e-8365-b379c0d28627)

## Actual behavior

The `end` argument isn't underlined as being repeated. Pylance does appear to work, as shown by the highlight of the missing closing paren:

![image](https://github.com/microsoft/pylance-release/assets/33569/ca34bb66-d1f3-449c-a3f4-d6d0027ea576)

## Logs

[pylance_log.txt](https://github.com/user-attachments/files/16024125/pylance_log.txt)

**Comment by erictraut** [comment]
An error is generated by this condition if you enable type checking. I'm guessing that you have `typeCheckingMode` set to "off". If you set it to "basic", you will see an error for the above code. If you don't want to enable all basic type checking rules, you could specifically enable the `reportCallIssue` diagnostic check.

I'm going to close the issue because pyright and pylance are working correctly here.
